tests/integration/cases/run_macaron_sh_script_unit_test/test_run_macaron_sh.py

```diff
@@ -1,17 +1,16 @@
-# Copyright (c) 2023 - 2023, Oracle and/or its affiliates. All rights reserved.
+# Copyright (c) 2023 - 2024, Oracle and/or its affiliates. All rights reserved.
 # Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/.
 
 """Tests for the ``run_macaron.sh`` script."""
 
 import os
-import subprocess
+import subprocess  # nosec
```

Comment:
```suggestion
import subprocess  # nosec B404
```

Reply:
Addressed in https://github.com/oracle/macaron/pull/782/commits/df0a4a92b9acebccd96f8b0a34af251b324cdf59